cogboard-webapp/src/components/Widget/index.js

```diff
@@ -176,6 +182,7 @@ const Widget = ({ id, index }) => {
             isEmptyHeader={isEmptyHeader}
             avatar={
               !expandContent &&
+              type !== 'ZabbixWidget' &&
```

Comment:
How many identical strings does it take to create a constant?

Reply:
@szymon-owczarzak I can add a constant for this text, but where? In the component or in constant/index.js?

Reply:
i would say component

Reply:
Done